core/src/Platforms/iOS/TokenCacheAccessor.cs

```diff
@@ -97,11 +96,14 @@ private string GetTeamId()
                 match = SecKeyChain.QueryAsRecord(queryRecord, out resultCode);
             }
 
+            if (resultCode != SecStatusCode.Success)
```

Comment:
I think it would be safer to also add OR match==null to the condition

Reply:
Also, do we want to add a bit of logging here?

Reply:
this should be removed...maybe it's showing from an earlier commit?